miniwob/scripts/gen_env_classes.py

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env python
+"""Generate environment classes and their docstrings."""
+import argparse
+import re
+from typing import Sequence
+
+import gymnasium
+
+CLASS_TEMPLATE = """
+class {camel_name}Env(MiniWoBEnvironment):
+    \"\"\"
+    ## Description
+
+    {desc}
+
+    ## Example utterances
+
+    {utterances}
+
+    ## Utterance fields
+
+    {fields}
+
+    ## Custom settings
+
+    None
+    \"\"\"
+
+    subdomain = "{name}"
+"""
+
+
+def _raw_env_name(env_id: str) -> str:
+    """Convert ID like miniwob/click-test-v1 to click-test."""
+    match = re.search("^miniwob/(.*)-v[0-9.]*$", env_id)
```

Comment:
Optional, you can use `gymnasium.envs.registration.parse_env_id` which will return the namespace, env_name and version then you just need to use the env_name variable

Reply:
Done.